body/stretch_body/wacc.py

```diff
@@ -32,15 +32,17 @@ class WaccBase(Device):
 
     TRIGGER_BOARD_RESET = 1
 
-    def __init__(self, ext_status_cb=None, ext_command_cb=None):
+    def __init__(self, ext_status_cb=None, ext_command_cb=None, usb=None):
         Device.__init__(self, 'wacc')
         self.ext_status_cb=ext_status_cb
         self.ext_command_cb=ext_command_cb
         self.lock=threading.RLock()
         self._dirty_config = True #Force push down
         self._dirty_command = False
         self._command = {'d2':0,'d3':0, 'trigger':0}
-        self.transport = Transport(usb='/dev/hello-wacc', logger=self.logger)
+        if usb is None:
+            usb='/dev/hello-wacc'
```

Comment:
Instead of hard-coding the usb alias here, let's make it a parameter of "wacc" in the robot_params_*_.py dictionaries. This gives users the ability to override in user YAML.

Reply:
It is a good idea and has been implemented with all the devices.